File: net/Link.Message.Client/content/VoiceMessageContent.cs
namespace Link.Message.Client.content
{

	/// <summary>
	/// 语音消息内容
	/// </summary>
	public class VoiceMessageContent : FileMessageContent
	{

		public VoiceMessageContent(string downloadUrl, int size, string fileName, string extension) : base(downloadUrl, size, fileName, extension)
		{
		}

		public VoiceMessageContent(string downloadUrl, int size, string fullFileName) : base(downloadUrl, size, fullFileName)
		{
		}
	}

}

File: java/link-message-client/src/main/java/link/message/client/content/VoiceMessageContent.java
package link.message.client.content;

/**
 * 语音消息内容
 * @author zhongt
 *
 */
public class VoiceMessageContent extends FileMessageContent {
	
	public VoiceMessageContent(String downloadUrl, int size, String fileName, String extension) {
		super(downloadUrl, size, fileName, extension);
	}
	
	public VoiceMessageContent(String downloadUrl, int size, String fullFileName) {
		super(downloadUrl, size, fullFileName);
	}
}
